lib/AlgorandTxHelper.mjs

```diff
@@ -0,0 +1,107 @@
+import algosdk from 'algosdk'
+import { MAX_UINT64 } from './validators/AlgorandAppTxValidator.mjs'
+
+export default class AlgorandTxHelper {
+  constructor(algodClient) {
+    this.algodClient = algodClient
+  }
+
+  async create(tx) {
```

Comment:
Thanks for the idea. This implementation is common for few projects so I would like to keep it inside one file. But if size of file is increased it will required update.